### PR TITLE
make animatable

### PR DESCRIPTION
**Pull request by morethanreal**
PTAL @cdata cc @notwaldorf

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
   "dependencies": {
     "polymer": "Polymer/polymer#v0.9.0-rc.1",
     "paper-dialog-behavior": "PolymerElements/paper-dialog-behavior#^0.9.0",
-    "paper-styles": "PolymerElements/paper-styles#^0.9.0"
+    "paper-styles": "PolymerElements/paper-styles#^0.9.0",
+    "neon-animation": "PolymerElements/neon-animation#^0.0.1"
```
Comment (cdata) [inline bower.json]: I don't think this release has actually been tagged yet.

Comment (cdata) [inline bower.json]: Also, what do you think about making the release line up with the other element catalogues? e.g., 0.9.x for Polymer@0.9.x.

Comment (morethanreal) [inline bower.json]: Do you mean the `version` field here to match the Polymer version?

File: bower.json [hunk 2]
```diff
@@ -20,7 +20,8 @@
   "dependencies": {
     "polymer": "Polymer/polymer#v0.9.0-rc.1",
```
Comment (notwaldorf) [inline bower.json]: I think we can use `#^0.9.0` for polymer now.

File: bower.json (hunk 1 above)
Comment (notwaldorf) [inline bower.json]: Should this be `PolymerElements/neon-animation#^0.9.0`?

File: paper-dialog.html [hunk 3]
```diff
@@ -44,8 +46,36 @@
...
+        this.backdropElement.close();
+      }
+      this.playAnimation('exit');
+    },
+
+    _onNeonAnimationFinish: function() {
+      if (this.opened) {
+        this._finishRenderOpened();
+      } else {
+        this._finishRenderClosed();
```
Comment (notwaldorf) [inline paper-dialog.html:76]: I like how simple these 3 functions look! :heart_eyes_cat:

**Comment by notwaldorf** [comment]
LGTM!

**Pull request merged**